modules/enable-active-backup-mode.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * networking/advanced_networking/network-bonding-considerations.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enable-active-backup-mode_{context}"]
+= Enable active-backup mode for your cluster
+
+[role="_abstract"]
+The `active-backup` mode provides fault tolerance for network connections by switching to a backup link where the primary link fails. The mode specifies the following ports for your cluster:
```

Comment:
[Avoid self-referential language in abstracts.](https://docs.google.com/document/d/1xRc_licMTbLrLG9vmJ4bP9aPk6mzwwly5lGlkRz9d9s/edit?tab=t.0)

Should we remove the second sentence so that it doesn't get added to the short desc?
```suggestion
[role="_abstract"]
You can use the `active-backup` mode to provide fault tolerance for network connections by switching to a backup link where the primary link fails. 

The mode specifies the following ports for your cluster:
```